src/api/spec/features/webui/image_templates_spec.rb

```diff
@@ -1,7 +1,4 @@
 require 'browser_helper'
-# WARNING: This test require real backend answers for projects/packages, make
```

Comment:
Are these all? I think in the meantime we got more of these ...

Reply:
yeah, but henne only wanted to remove them from features tests. I'll remove the rest later on :)

Reply:
Fine for me